voila/static_file_handler.py

```diff
@@ -32,7 +32,7 @@ def initialize(self, paths, default_filename=None):
     def get_absolute_path(self, root, path):
         # find the first absolute path that exists
         self.root = self.roots[0]
-        for root in self.roots:
+        for root in self.roots:git 
```

Comment:
I'm guessing this shouldn't be here 😉

Reply:
hahahah I do this all the time

Reply:
🤦‍♂

Reply:
😆

Reply:
Just be glad it wasn't a password (right??) 😄